app/src/main/java/ch/epfl/sdp/contamination/fragment/UserInfectionFragment.java

```diff
@@ -253,7 +263,7 @@ private void setInfectionColorAndMessage(boolean infected) {
 
     private void clickAction(Button button, TextView textView, int buttonText, int textViewText, int textColor) {
         button.setText(buttonText);
-        textView.setTextColor(getResources().getColorStateList(textColor, requireActivity().getTheme()));
+        textView.setTextColor(getResources().getColorStateList(textColor, getActivity().getTheme()));
```

Comment:
We should add an if/else condition or a try/catch block to handle the potential nullPointerException. 
And could we retrieve only once the context when initialising the fragment?

Reply:
We could, but since that function is executed only when a button is clicked I think it's safe not to catch that